### DISCUSSION TITLE
What is the unit of output wavefield?

### DISCUSSION THREAD

**Discussion by tjk9501** [slimgroup/JUDI.jl#116] [Q&A] [answered]
Hello, Dr. Louboutin

What is the unit of JUDI's output wavefield and observed time data? I wonder if the u(x,t) in this page
```
https://wiki.seg.org/wiki/Full-waveform_inversion,_Part_1:_Forward_modeling
```
denotes displacement field (whose unit is mm when I am using a unit system of mm, g, us), or u(x,t) is the pressure field (whose unit is something like Pa)?

Thanks!

**Comment by mloubout** [comment, marked as answer]
The Wakefield is the pressure as JUDI currently only consider variations of th3 acoustic wave equation.

**Reply by tjk9501** [reply]
Thanks!!!!!!!!
